js/src/base-component.js

```diff
@@ -61,6 +61,12 @@ class BaseComponent {
     return Data.get(element, this.DATA_KEY)
   }
 
+  static getOrCreateInstance(element, config = {}) {
+    const _config = typeof config === 'object' ? config : null
+
+    return this.getInstance(element) || new this(element, _config)
+  }
+
```

Comment:
```suggestion
  static getOrCreateInstance(element, config = {}) {
    return this.getInstance(element) || new this(element, typeof config === 'object' ? config : null)
  }
```

To prevent the check if the instance already exists.